README.md

```diff
@@ -1 +1,28 @@
-# eq-smoke-tests
\ No newline at end of file
+# eq-smoke-tests
+
+## Background
+
+This repo is a simple repeatable smoke test that verifies the eQ services work together
+So that when new versions of service are deployed we can be sure we haven't broken anything.
+
+## PREREQUISITE
+Oracle JDK v1.8+
+You can check your Java version in the terminal with java -version (single dash)
+
+Be sure to install the JDK not the JRE.
+
+## Installation
+```
+npm install chimp
+```
+
+## How to run
+To run on eq-compose locally:
+```
+BASE_URL=http://localhost:3000/ yarn run chimp config/chimp.smoke.js
```

Comment:
Is there a default naming or location for the config so we dont have to override it in this command?